pytest_doctestplus/plugin.py

```diff
@@ -410,6 +414,14 @@ def get_list_opt(name):
                 self._ignore_paths.append(path)
                 break
 
+        for option in config.getini("doctest_subpackage_requires"):
+            subpackage_pattern, required = option.split('=', 1)
```

Comment:
Will adding spaces around `=` affect the logic here?

Reply:
I'll check!

Reply:
Yep it works! But I added calls to strip() in my latest commit to make *really* sure.